src/sql/magic.py

```diff
@@ -191,6 +193,16 @@ def __init__(self, shell):
         type=str,
         help="Assign an alias to the connection",
     )
+    # @argument("-G", "--use-globals", action="store_true", help="use global namespace")
```

Comment:
let's remove this for now as we don't know if we'll immediately add this

Reply:
now remove in https://github.com/ploomber/jupysql/commit/143b1af0d9a91c88fd6596628e9eb7f97198e80b